src/bindgen/ir/ty.rs

```diff
@@ -64,39 +64,69 @@ impl PrimitiveType {
             "f32" => PrimitiveType::Float,
             "f64" => PrimitiveType::Double,
 
-            _ => {
-                let (kind, signed) = match path {
-                    "c_short" => (IntKind::Short, true),
-                    "c_int" => (IntKind::Int, true),
-                    "c_long" => (IntKind::Long, true),
-                    "c_longlong" => (IntKind::LongLong, true),
```

Comment:
This PR is best reviewed with whitespace changes hidden:
![image](https://github.com/mozilla/cbindgen/assets/9323865/1d881458-f376-4358-9ba2-1df82965ac28)